<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1671</width>
    <height>999</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>498</width>
            <height>230</height>
           </size>
          </property>
          <property name="title">
           <string>Główne opcje</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <property name="spacing">
              <number>13</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <widget class="QPushButton" name="START_Button">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>START</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="STOP_Bttun">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>STOP</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="RESET_Button">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>RESET</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Okno czasowe</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBoxOknoczasowe">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>5</number>
                 </property>
                 <property name="maximum">
                  <number>50</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>14</number>
               </property>
               <item>
                <widget class="QPushButton" name="Konf_ARX_Button">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Konfiguracja ARX</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QPushButton" name="Wczytaj_Button">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Wczytaj</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Zapisz_Button">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Zapisz</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Interwał</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinBOX_Interwal">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="decimals">
                  <number>0</number>
                 </property>
                 <property name="minimum">
                  <double>10.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>50.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>199</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>sygnał</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="Sin_Button">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>sinus</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Square_Button">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>square</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="sizeIncrement">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="lineWidth">
                    <number>1</number>
                   </property>
                   <property name="text">
                    <string>amplituda</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="spinBOX_Amplituda">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <double>1000.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                   <property name="value">
                    <double>1.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Okres</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="spinBOX_Czstotliwosc">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximum">
                    <double>1000.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.010000000000000</double>
                   </property>
                   <property name="value">
                    <double>3.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_15">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Wypełnienie P</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="spinBox_Wypelnienie">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximum">
                    <double>1.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>0.500000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_16">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Stała S</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="SpinBox_Stala">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="minimum">
                    <double>-1000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="maximumSize">
             <size>
              <width>199</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>PID</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>rózniczkowania Td</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QPushButton" name="Reset_d">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>RESET d</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="spinBOX_Td">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="maximum">
                      <double>1000.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>0.100000000000000</double>
                     </property>
                     <property name="value">
                      <double>0.200000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>całkowania Ti</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QPushButton" name="Reset_i">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string> RESET I</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="spinBOX_Ti">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="maximum">
                      <double>1000.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>0.100000000000000</double>
                     </property>
                     <property name="value">
                      <double>5.000000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_20">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>wzmocnienie K</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="spinBOX_WzmocK">
                     <property name="minimumSize">
                      <size>
                       <width>82</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="decimals">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <double>1000.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>0.100000000000000</double>
                     </property>
                     <property name="value">
                      <double>0.500000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QRadioButton" name="radio_przed">
                     <property name="text">
                      <string>przed suma</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">buttonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radio_pod">
                     <property name="text">
                      <string>pod suma</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">buttonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1671</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUAR">
    <property name="title">
     <string>UAR</string>
    </property>
   </widget>
   <addaction name="menuUAR"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
